src/components/header.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import PropTypes from 'prop-types';
+
+const Header = ({ siteTitle }) => (
+  <header
+    style={{
+      background: 'rebeccapurple',
```

Comment:
I'd prefer not to do styles inline to the extent that we can. If we're going to use Material UI, let's use something like Material UI's [styles API](https://material-ui.com/styles/basics/) to do this.

Reply:
Go it :)

I installed `styled-components` ([link](https://www.styled-components.com/)). It is like Material UI ... 
If you prefer Material UI I can change